### PR TITLE
Make error-page-embed.js lintable; fix lint failures

### PR DESCRIPTION
**Pull request by benvinegar**

### CODE REVIEW

File: src/sentry/templates/sentry/error-page-embed.js [hunk 1]
```diff
@@ -17,20 +19,15 @@
...
   var XHR_DONE = 4;
 
-  var template = {{ template }};
-  var endpoint = {{ endpoint }};
-  var encode = window.encodeURIComponent;
-
-  var escape = function(str) {
-    return str.replace(/&/g,'&amp;').replace(/</g,'&lt;').replace(/>/g,'&gt;');
-  };
+  var template = /*{{ template }}*/'';
```
Comment (mattrobenolt) [inline src/sentry/templates/sentry/error-page-embed.js]: This is really weird. I see what it's doing, but is this a common pattern?

File: src/sentry/web/frontend/error_page_embed.py [hunk 2]
```diff
@@ -111,8 +111,8 @@ def dispatch(self, request):
         })
 
         context = {
-            'endpoint': mark_safe(json.dumps(request.get_full_path())),
-            'template': mark_safe(json.dumps(template)),
+            'endpoint': mark_safe('*/' + json.dumps(request.get_full_path()) + ';/*'),
+            'template': mark_safe('*/' + mark_safe(json.dumps(template)) + ';/*'),
```
Comment (mattrobenolt) [inline src/sentry/web/frontend/error_page_embed.py]: Do you need two `mark_safe`s here? I feel like you only need the one.

File: src/sentry/templates/sentry/error-page-embed.js (hunk 1 above)
Comment (benvinegar) [inline src/sentry/templates/sentry/error-page-embed.js]: Common – no. But we did it at Disqus for a couple things like this.

File: src/sentry/web/frontend/error_page_embed.py (hunk 2 above)
Comment (benvinegar) [inline src/sentry/web/frontend/error_page_embed.py]: Good catch, thanks.

**Pull request merged**